CONTRIBUTING.md

```diff
@@ -91,25 +91,73 @@ it will create a new folder inside `packages` with a simple boilerplate for your
 
 ### Prerelease
 
-- Every merged PR should trigger an automatic canary release
-- The version number is calculated by a combination of the output of `git describe` and the `publish:prerelease` script in the root package.json. Specifically having `major` is appropriate for the pre-7.0 development phase, but we should change it to `minor` after 7.0.0 is finally released.
+A [prerelease](https://github.com/Turfjs/turf/blob/master/.github/workflows/prerelease.yml) action is available that publishes a canary release for every commit or PR merged to the master branch.  However, this action is only [enabled](https://github.com/Turfjs/turf/actions/workflows/prerelease.yml) when needed.
+
+When used, it publishes an alpha release to NPM (e.g. `7.0.0-alpha.116` where 116 is the number of commits to master since the last release tag).
+- The version number is calculated by a combination of the output of `git describe` and the `publish:prerelease` script in the root package.json. It is typically setup to do a `minor` prerelease, but can be changed, such as prior to a `major` release.
 
 ### Release
-- If necessary, make and merge a PR to contain any last minute changelog entries or other housekeeping items
-- Make a PR that contains the file changes from `pnpm lerna version --no-push 7.0.0` (update the version as necessary)
-- When pushing the release PR, use `--follow-tags` to make sure the relevant tag is also pushed
-- The CI will start doing the release when the tag is created
-- Don't forget to merge your PR
+
+A turf release is done from your local computer using a clean, up-to-date version of the turf master branch.
+
+- If necessary, make and merge a PR with any last minute housekeeping items.
+  - Turf's documentation (README.md files) should be up to date as they are updated automatically on commit, based on the JSDoc comments in the package files.  See the [documentation](#documentation) for how to regenerate them if needed.
+- Review code commits and decide the new version number to be published (for example 7.0.1 or 7.1.0).
+  - If there are breaking changes, then it should be a major version bump, e.g. 7.x.x to 8.0.0.  This project follows [semantic versioning](https://semver.org/).
+- When running release commands below, replace `7.0.0` with the version number you are releasing.  Now, let's get started.
+
+- fetch the latest code from remote origin
```

Comment:
Do you think there is the potential for confusion and that someone might try this in their development fork? Should we specify cloning a new local repo?

```bash
git clone git@github.com:Turfjs/turf.git turf-release
cd turf-release/
```

Reply:
Cloning a new local repo for a release seems like overkill (open to pushback).  But I understand the desire to make sure they have a fresh local copy of the right remote.  I am working on more specific steps for cloning and setting up your repo right now.  Let me take a shot at it.

Reply:
Fwiw, I wouldn't bat an eye at having to clone a fresh repo for a release. It takes one minute, 100MB of disk space, and eliminates any potential for confusion or cruft from previous builds. Worth the cost to benefit for something we're probably only doing once a month.

Reply:
Is there anything more to be done if working with a fresh install?  `pnpm install` at least?

Reply:
Lucky you got me to check. Yes, lerna needs nx installed, so it would be an `pnpm install`. Which builds the whole project unnecessarily, but I don't know a way around that at the moment.

Reply:
Thanks.  I'll move towards checking some of this

Reply:
I included the fresh clone steps as you suggested.  As a follow-on to that I included how to work with an existing copy, with extra cleanup/verify steps.